src/services/make.service.ts

```diff
@@ -19,3 +20,31 @@ export async function runSavePDFScenario() {
     return { error };
   }
 }
+
+export async function sendMessageWebhook(bookingInfo: Booking, boatDetails: Boat) {
+  try {
+    const queryParams = new URLSearchParams({
+      date: moment(bookingInfo.Date).format("DD/MM/YY"),
+      id: bookingInfo.id,
+      firstName:bookingInfo["First Name"],
+      lastName:bookingInfo["Last Name"],
+      customerEmail: bookingInfo.Email,
+      boatName: boatDetails.Nombre,
+      pricePerMile: (boatDetails.MilePrice ?? "0").toString(),
+      totalPassengers: (bookingInfo["Total Passengers"] ?? "0").toString(),
+    }).toString();
```

Comment:
Here would had being better send as a body just the full bookingInfo and boat details.  - I wonder you tried but wasn't working

But i think is ok for now